go-controller/pkg/ovn/gress_policy.go

```diff
@@ -45,24 +45,54 @@ type portPolicy struct {
 	port     int32
 }
 
-func (pp *portPolicy) getL4Match() (string, error) {
+func (pp *portPolicy) getL4Match(protoFilters map[string]string) error {
```

Comment:
nit: naming of this function is now a little confusing. It is now appending new matches instead of just "get". So I suggest to rename the function to appendL4Match. In addition, the param protoFilters is better to match the naming of the function, e.g. l4Matches.